fr/releases/centreon-commercial-extensions.md

```diff
@@ -293,6 +293,13 @@ liens
 
 ## Centreon Auto Discovery
 
+### 20.04.8
+
```

Comment:
Missing this ticket : [HD/Provider/Install/Upgrade] Limit provider definition version that is imported to 2.0 in 20.10/20.04